templates/none/routes/index.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express');
+var router = express.Router();
+
+router.get('/', function (req, res, next) {
+  res.sendFile('index.html', {
```

Comment:
I don't have much of an opinion - I'm just starting in with Express, so I was trying to mimic the existing view engines. If using `express.static` this could just be folded into `app.js`, right?